src/magic.cpp

```diff
@@ -2742,13 +2742,15 @@ void spellcasting_callback::display_spell_info( size_t index )
         if( !sp.components().get_components().empty() ) {
             for( const std::string &line : sp.components().get_folded_components_list(
                      width - 2, c_light_gray, pc.crafting_inventory( pc.pos(), 0, false ), return_true<item> ) ) {
-                info_txt.emplace_back( line );
+                ImGui::TextWrapped("%s", remove_color_tags(line).c_str());
```

Comment:
Since you’re no longer using the `info_txt` property, go ahead and remove it from the class as well.

Reply:
removed it along with another unused property 👍